src/modules/account/components/Overview/overview.js

```diff
@@ -1,21 +1,17 @@
-import React, { useState, useRef, useEffect } from 'react';
+/* eslint-disable max-statements */
```

Comment:
Can we move these just above the head of the component keeping the import section clean